scheduler/scheduler-server/src/test/java/functionaltests/RegressionTestSuite.java

```diff
@@ -0,0 +1,96 @@
+import functionaltests.db.schedulerdb.TestJobRemove;
+import functionaltests.db.schedulerdb.TestMultipleTasks;
+import functionaltests.job.taskkill.TestJobKilled;
+import functionaltests.job.taskkill.TestProcessTreeKiller;
+import functionaltests.recover.TaskReconnectionWithForkedTaskExecutorTest;
+import functionaltests.recover.TaskReconnectionWithInProcessTaskExecutorTest;
+import functionaltests.recover.TaskRecoveryWhenNodesAreReservedInBatchTest;
+import functionaltests.rm.TestOperationsWhenUnlinked;
+import functionaltests.workflow.TestJobLegacySchemas;
+import functionaltests.workflow.TestWorkflowSubmission;
+import functionaltests.workflow.complex.TestWorkflowComplexJobs;
+import functionaltests.workflow.complex.TestWorkflowComplexJobs2;
+import functionaltests.workflow.complex.TestWorkflowComplexJobs3;
+import functionaltests.workflow.complex.TestWorkflowComplexJobs4;
+import functionaltests.workflow.complex.TestWorkflowComplexJobs5;
+import functionaltests.workflow.complex.TestWorkflowIfJobs;
+import functionaltests.workflow.complex.TestWorkflowLoopJobs;
+import functionaltests.workflow.complex.TestWorkflowLoopJobs2;
+import functionaltests.workflow.complex.TestWorkflowReplicateJobs;
+import functionaltests.workflow.complex.TestWorkflowReplicateJobs2;
+import functionaltests.workflow.complex.TestWorkflowReplicateJobs3;
+
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+                      // DB Tests
+                      SchedulerDbManagerConcurrencyTest.class, TestJobAttributes.class, TestJobRemove.class,
+                      TestMultipleTasks.class,
+
+                      // misc
+                      TestOperationsWhenUnlinked.class,
+
+                      // Tests without scheduler restart
+                      TestJobInstantGetTaskResult.class, TestMultipleUsersMakingMassivGetResultRequest.class,
+                      TestJobLegacySchemas.class,
+
+                      // Non forked mode no restart
+                      TestWorkflowSubmission.class, TestWorkflowComplexJobs.class, TestWorkflowComplexJobs2.class,
+                      TestWorkflowComplexJobs3.class, TestWorkflowComplexJobs4.class, TestWorkflowComplexJobs5.class,
+                      TestWorkflowIfJobs.class, TestWorkflowLoopJobs.class, TestWorkflowLoopJobs2.class,
+                      TestWorkflowReplicateJobs.class, TestWorkflowReplicateJobs2.class,
+                      TestWorkflowReplicateJobs3.class,
+
+                      // Tests with scheduler restart
+                      TestProcessTreeKiller.class,
+
+                      // Tests custom configuration
+                      TestJobKilled.class,
+
+                      TestDataspaceConcurrentTransfer.class, TestDataspaceConcurrentKilling.class,
+                      TaskReconnectionWithForkedTaskExecutorTest.class,
+                      TaskReconnectionWithInProcessTaskExecutorTest.class,
+                      TaskRecoveryWhenNodesAreReservedInBatchTest.class })
```

Comment:
`TaskReconnectionWithForkedTaskExecutorTest.class`, `TaskReconnectionWithInProcessTaskExecutorTest.class`, and `TaskRecoveryWhenNodesAreReservedInBatchTest.class` are tests with scheduler restart. We can move them to the `// Tests with scheduler restart` category above.

Reply:
I put them in the category "with custom configuration"